### DISCUSSION TITLE
Hellp litmatica

### DISCUSSION THREAD

**Discussion by Evgeha69** [maruohon/litematica#882] [Q&A]
The stick does not work and because of this the modes are not displayed. The stick is set in the settings

**Comment by Samalando** [comment]
Только на английском языке и pojav launcher не поддерживается (google translate в помощь)

**Comment by benhovinga** [comment]
It may be because your rendering is turned off. The default key bind is `M + R` to enable/disable rendering. When the rendering is turned off the stick is just a regular stick.
